src/common/KokkosKernels_Handle.hpp

```diff
@@ -195,6 +195,7 @@ class KokkosKernelsHandle
   typedef typename size_type_persistent_work_view_t::HostMirror size_type_persistent_work_host_view_t; //Host view type
   typedef typename Kokkos::View<nnz_scalar_t *, HandleTempMemorySpace> scalar_temp_work_view_t;
   typedef typename Kokkos::View<nnz_scalar_t *, HandlePersistentMemorySpace> scalar_persistent_work_view_t;
+  typedef typename Kokkos::View<nnz_scalar_t **, Kokkos::LayoutLeft, HandlePersistentMemorySpace> scalar_persistent_work_view2d_t;
```

Comment:
@mhoemmen Which operator= are you talking about? KokkosKernels::Handle doesn't have an explicit one (well, it looks like it did but it has been commented out). I don't think there's ever an occasion where you'd want to copy a handle directly.

Reply:
Oops, my bad, it's (sort of?) a copy constructor.  (I say "sort of" because it takes the right-hand side by nonconst reference, yet it looks like the intent is not to take ownership from the right-hand side.)

Reply:
@mhoemmen Oh I see. I think all the Handle getters are const (or, they should be) so I'll try to pass ``right_side_handle`` by const&.

Reply:
@mhoemmen I wasn't able to use const&, since the subhandle getters aren't const.

Reply:
@brian-kelley Thanks for trying!  It's OK to leave that broken for later :-)